front/preface.tex

```diff
@@ -17,4 +17,4 @@
 
 \hfill \break
 
-Sondre Lefsaker and Håvard Wormdal Høiby \textbox{\hfill 12.06.2015}
```

Comment:
For me this makes the date float to the right. If it does so without the `textbox` for you, thats ok.